addons/account/wizard/account_move_send.py

```diff
@@ -303,6 +304,15 @@ def _compute_mail_attachments_widget(self):
             else:
                 wizard.mail_attachments_widget = []
 
+    @api.depends('move_ids')
+    def _compute_sequence_gap_warning(self):
+        for wizard in self:
+            gap_moves = wizard.move_ids.check_move_sequence_chain()
+            if not gap_moves:
+                wizard.sequence_gap_warning = _("Sending these invoices will create a gap in the sequence.")
```

Comment:
I'm not sure I understand myself what this is supposed to mean?
How is the move sequence related to the hashing?

Why would something that is not the last move(s) in the sequence trigger a warning when being sent?

Reply:
I think the idea was that we wanted to warn them of a broken sequence before hashing the move @Olivier-olma what do you think?

Reply:
Fair, but it needs a separate commit then